docs/resources/policy_exception.md

```diff
@@ -27,13 +27,36 @@ resource "lacework_policy_exception" "example" {
 }
 ```
 
+
+Create a Lacework Policy Exception to exempt specified resourceTags from policy.
+
+```hcl
+resource "lacework_policy_exception" "example" {
+  policy_id   = "lacework-global-73"
+  description = "Exception for resource tag example1 and example2"
+
+  constraint {
+    field_key = "resourceTags"
+    field_value_map {
+      key   = "example_tag1"
+      value = ["example_value", "example_value1"]
+    }
+    field_value_map {
+      key   = "example_tag2"
+      value = ["example_value", "example_value1"]
+    }
+  }
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
 * `description` - (Required) The description of the policy exception.
 * `policy_id` - (Required) The id of the policy the exception is associated.
 * `constraint` - (Required) Constraint. See [Constraint](#Constraint) below for details.
+* `field_value_map` - (Optional) FieldValueMap. See[FieldValueMap](#FieldValueMap) below for details.
```

Comment:
This doesn't go here. Move it to the next section (Constraint)

Reply:
👍 resolved in https://github.com/lacework/terraform-provider-lacework/pull/619/commits/d7d38e17661cb36798815e376acfadda914d6560